File: FrameIncam.Domains/Models/Master/Geo/MasterGeoType.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace FrameIncam.Domains.Models.Master.Geo
{
    [Table("master_geo_type")]
    public class MasterGeoType : Model
    {
        [Column("Geo_Type_Code")]
        public int? GeoTypeCode { get; set; }
        [Column("Geo_Type_Name")]
        public string GeoTypeName { get; set; }
        [Column("Geo_Type_Isactive")]
        public int? GeoTypeIsactive { get; set; }
    }
}

File: FrameIncam.Client.FreeLancer/ng/domains/master/geo/masterGeoType.js
(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var geo = p_models.namespace("fincam.master.geo");

            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: geo,
                name: "masterGeoType",
                base: {
                    namespace: fincam,
                    name: "model"
                },
                properties: {
                    id:  "int",
                    geoTypeCode: "int",
                    geoTypeName:  "string",
                    geoTypeIsactive: "int"
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));